pkg/partials/partials_test.go

```diff
@@ -0,0 +1,72 @@
+package partials
+
+import (
+	"encoding/json"
+	"errors"
+	"testing"
+
+	sdkerror "cosmossdk.io/errors"
+	"github.com/stretchr/testify/require"
+)
+
+// TODO(@h5law): Expand coverage with more test cases when more request types
+// are implemented in the partials package.
+func TestPartials_GetErrorReply(t *testing.T) {
```

Comment:
Make sure to rename `Partials` depending on how we call it per the discussion above